back-end/services/checkReasons.service.js

```diff
@@ -0,0 +1,93 @@
+export const PRIVACY_KEYWORDS = ["provide", "email", "phone", "mobile number",
+    "location", "credit card", "driver's licence no.", "driver's licence number"];
+
+export const NO_SOLUTION_KEYWORDS = ["sorry", "can't help you", "no solution", "don't understand",
```

Comment:
I think we should add shorter keywords (i.e., just one word instead of a phrase). We can't guarantee that a chatbot will use a specific combination of words.

For example, if we have the keyword "customer service rep", we could also add "customer service", "representative", ...)

Also, we might consider changing "rep" to "representative" or we could add another keyword with the full word.

Reply:
ok